setup/crm_phonecall/odoo/addons/crm_phonecall/crm_phonecall

```diff
@@ -0,0 +1 @@
+../../../../crm_phonecall
```

Comment:
Mmmh, I think your link is not good (see Travis error). Did you use setuptools-odoo-make-default ?

Reply:
i was getting this erreur when i do it : 
OSError: [WinError 1314] Le client ne dispose pas d'un privilège nécessaire: '..\\..\\..\\..\\crm_phonecall' -> '.\\setup\\crm_phonecall\\odoo\\addons\\crm_phonecall'
i tried with administrator same thing.